Auth0Tests/CredentialsManagerSpec.swift

```diff
@@ -454,8 +477,99 @@ class CredentialsManagerSpec: QuickSpec {
+                        }
+                    }
+                }
+
+                it("should yield a new access token with a new scope") {
+                    credentials = Credentials(accessToken: AccessToken, refreshToken: RefreshToken, expiresIn: Date(timeIntervalSinceNow: ExpiresIn), scope: "openid profile")
+                    _ = credentialsManager.store(credentials: credentials)
+                    waitUntil(timeout: 2) { done in
+                        credentialsManager.credentials(withScope: "openid profile offline_access", minTTL: 0) { error, newCredentials in
+                            expect(error).to(beNil())
+                            expect(newCredentials?.accessToken) == NewAccessToken
+                            done()
+                        }
+                    }
+                }
+
+                it("should not yield a new access token with a min ttl less than its expiry") {
+                    credentials = Credentials(accessToken: AccessToken, refreshToken: RefreshToken, expiresIn: Date(timeIntervalSinceNow: ExpiresIn))
+                    _ = credentialsManager.store(credentials: credentials)
+                    waitUntil(timeout: 2) { done in
+                        let ttl = Int((ExpiresIn - 1000) / 1000)
+                        credentialsManager.credentials(withScope: nil, minTTL: ttl) { error, newCredentials in
+                            expect(error).to(beNil())
+                            expect(newCredentials?.accessToken) == AccessToken
+                            done()
+                        }
+                    }
+                }
+
+                it("should yield a new access token with a min ttl greater than its expiry") {
+                    credentials = Credentials(accessToken: AccessToken, refreshToken: RefreshToken, expiresIn: Date(timeIntervalSinceNow: ExpiresIn))
+                    _ = credentialsManager.store(credentials: credentials)
+                    waitUntil(timeout: 2) { done in
+                        let ttl = Int((ExpiresIn + 1000) / 1000)
+                        credentialsManager.credentials(withScope: nil, minTTL: ttl) { error, newCredentials in
+                            expect(error).to(beNil())
+                            expect(newCredentials?.accessToken) == NewAccessToken
+                            done()
+                        }
+                    }
+                }
+
+                it("should fail to yield a renewed access token with a min ttl greater than its expiry") {
+                    credentials = Credentials(accessToken: AccessToken, refreshToken: RefreshToken, expiresIn: Date(timeIntervalSinceNow: ExpiresIn))
+                    _ = credentialsManager.store(credentials: credentials)
+                    waitUntil(timeout: 2) { done in
+                        credentialsManager.credentials(withScope: nil, minTTL: 100) { error, newCredentials in
+                            expect(error).toNot(beNil())
```

Comment:
let's cover the error message as well

Reply:
Thanks, changed in https://github.com/auth0/Auth0.swift/pull/399/commits/04ffa990458113d1f2963b18bb82d2fc82d062c2.